Jenkinsfile

```diff
@@ -7,7 +7,7 @@
 * Jenkins populates the patch version depending on the branch.
 */
 
-String VERSION = "2.3"
+String VERSION = "2.3.1"
```

Comment:
The patch version is set by Jenkins, so this should get a minor version bump instead:

```
String VERSION = "2.4"
```

Reply:
im getting error on running yarn test -u, parameter u not found

Reply:
I'll look into that; looks like you don't need to update any snapshots for this change anyway

Reply:
cool!